addons/website_sale/models/website.py

```diff
@@ -450,26 +450,6 @@ def sale_get_order(self, force_create=False):
     def _prepare_sale_order_values(self, partner_sudo):
         self.ensure_one()
         addr = partner_sudo.address_get(['delivery', 'invoice'])
```

Comment:
You can remove this line too (and the two values in the dict below) so that the standard `sale` compute methods handle the address computation.